site.html

```diff
@@ -0,0 +1,123 @@
+			</figure>
+			<figure>
+				<img src="images/Slider03.png" height= "250px" alt="Image-Slider03" />
+			</figure>
+		</section>
+		<section>
+			<h1>Serviços</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod diam eget ex rhoncus, sed.</p>
+			<hr />
+			<article>
+				<img src="images/Service1.png" alt="Service1" />
+				<h3>Websites</h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque.</p>
+			</article>
+			<article>
+				<img src="images/Service2.png" alt="Service2" />
+				<h3>Templates</h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque.</p>
+			</article>
+			<article>
+				<img src="images/Service3.png" alt="Service3" />
+				<h3>Temas WordPress</h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque.</p>
+			</article>
+		</section>
+		<section>
+			<h1>Meus Trabalhos</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed varius consectetur felis ut condimentum. Aliquam.</p>
+			<hr />
+			<nav>
+				<ul>
+					<li><a href="">Tudo</a></li>
+					<li><a href="">Website</a></li>
+					<li><a href="">Template</a></li>
+					<li><a href="">Tema</a></li>
+				</ul>
+			</nav>
+			<article>
+				<img src="images/Portfolio01.png" height= "250px" alt="Portolio01" />
+				<h3>Lorem ipsum</h3>
+				<p>Lorem ipsum...</p>
+			</article>
+			<article>
+				<img src="images/Portfolio02.png" height= "250px" alt="Portfolio02" />
+				<h3>Lorem ipsum</h3>
+				<p>Lorem ipsum...</p>
+			</article>
+			<article>
+				<img src="images/Portfolio03.png" height= "250px" alt="Portfolio03" />
+				<h3>Lorem ipsum</h3>
+				<p>Lorem ipsum...</p>
+		</section>
+		<section>
+			<h1>Quem sou eu</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean quis dui malesuada purus aliquet accumsan.</p>
+			<hr />
+			<figure>
+				<img src="images/photo.png" height= "500px" alt="Photo" />
+				<figcaption>Joana D'arc</figcaption>
+			</figure>
+			<div>
+				<h2>Sobre mim</h2>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur id felis vestibulum, laoreet ligula sed, fermentum libero. In blandit tristique leo id dictum. Nullam gravida vulputate ipsum. Nulla vel lacus sapien. Praesent dapibus mi justo, nec fringilla nisi congue quis. Suspendisse at sapien nec odio gravida placerat. Aliquam iaculis quis dui et ullamcorper. Vestibulum id dui tempus, feugiat velit non, bibendum leo. Nam in mollis libero. In sit amet diam in mauris sodales tincidunt et quis velit. Ut sed odio nec erat eleifend mollis vitae.</p>
+			</div>
+			<div>
+				<h2>Habilidades</h2>
+				<p>Java Script</p>
+				<p>Html & CSS</p>
+				<p>jQuery</p>
+				<p>Wordpress</p>
+			</div>		
+		</section>
+		<section>
+			<h1>Contato</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus id enim vitae quam suscipit porta.</p>
+			<hr />
+			<div>
+				<p>meuemail@mail.com</p>
```

Comment:
Você pode facilitar a vida do teu usuário colocando seu email clicável.

```suggestion
				<a href="mailto:meuemail@mail.com">meuemail@mail.com</a>
```